src/code/index.js

```diff
@@ -44,3 +44,27 @@ showBtn.addEventListener("click", () => {
         table.classList.add("hidden");
     }
 });
+
+function readSocialLinks() {
+    const socialLinks = {};
+
+    fetch("../social-links.json")
```

Comment:
return the Promise from the function from here so that I can use the data

Reply:
Already fixed, my bad.